apps/btshell/src/btshell.h

```diff
@@ -182,6 +185,36 @@ int btshell_l2cap_send(uint16_t conn, uint16_t idx, uint16_t bytes);
 int btshell_l2cap_reconfig(uint16_t conn_handle, uint16_t mtu,
                            uint8_t num, uint8_t idxs[]);
 
+#if (MYNEWT_VAL(BLE_ISO_BROADCASTER))
+int btshell_broadcast_base_add(uint8_t adv_instance, uint32_t presentation_delay);
```

Comment:
What this method is doing ? Isn't this a "StartBasicAudioAnnouncement" function ? not sure about having here presentation delay. Where is the "real" BASE prepared ? 

What's the flow ? How to start from scratch a broadcast ?
1. Who prepares Broadcast Audio Announcement
2. Who prepares Broadcast Source Audio Endpoint
3. Where it'll be attached to EA, PA
4. Where EA, PA would be started
5. Where data would be bind to BISes
?

Reply:
So the flow is like that:
1. `base_add` (calls `btshell_broadcast_base_add`) - allocates memory for `ble_audio_base`, `ble_iso_big_params`. Sets `presentation_delay` and `adv_instance` to base
2. `big_sub_add` (calls `btshell_broadcast_big_sub_add`) - adds subgroup to BASE identified by `adv_instance` (allocates memory, sets fields)
3. `bis_add` (calls `btshell_broadcast_bis_add`) - similar to `big_sub_add`, but for BIS. Adds bis to last added subgroup in BIG identified by `adv_instance`
4. `broadcast_create` (calls `btshell_broadcast_create` - basically calls `ble_audio_broadcast_create`. Here EA and PA params are set, instance configured with both advertisements.
5. `broadcast_start` (calls `btshell_broadcast_start`) - this finally calls `ble_audio_broadcast_start`, so EA, PA are started. `ble_iso_create_big` is also called, and BIG Info advertisements start to broadcast
6. In shell, I haven't added data stream flow to BISes. I'm not sure how this could be done in shell to make sense. I'm preparing standalone demo that does all this, in similar way extended_advertiser demo does. So at this point app has to have some routine (like loop or thread) that calls `ble_iso_tx`, with payload. For BIS `conn_handle` for this function is BIS handle.
7. At this point broadcast can be stopped (`broadcast_stop`), destroyed (`broadcast_destroy`) or updated (`broadcast_update`).